server/src/termdb.sql.js

```diff
@@ -103,25 +103,31 @@ return a sample count of sample names passing through the filter
 	}
 
 	const filter = await getFilterCTEs(j, ds)
-	let statement, row
-	let sample_type
 	if (ds.cohort.db.tableColumns['sampleidmap'].includes('sample_type')) {
-		statement = `WITH ${filter.filters}
+		const statement = `WITH ${filter.filters}
 		SELECT count (distinct sample) as count, sample_type
 		FROM ${filter.CTEname} join sampleidmap on sample = sampleidmap.id group by sample_type`
-		row = ds.cohort.db.connection.prepare(statement).get(filter.values)
-		sample_type = ds.cohort.termdb.sampleTypes[row.sample_type]
-		sample_type = row.count > 1 ? sample_type.plural_name : sample_type.name
+		const rows = ds.cohort.db.connection.prepare(statement).all(filter.values)
+		const text = [] 
+		for(const row of rows) {
+			const st = ds.cohort.termdb.sampleTypes[row.sample_type]
+			const name = row.count > 1 ? st.plural_name : st.name
+			text.push(`${row.count} ${name}`)
+		}
+		return {count: text.join('; ')}
```

Comment:
This wont happen, but if so, I was joining with and, see
<img width="202" alt="Screenshot 2025-02-19 at 2 49 11 PM" src="https://github.com/user-attachments/assets/123dcca7-6b61-49c9-9385-6f005d7ce09e" />